index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Alice Cieslik . Persönliche Website</title>
+  </head>
+  <body>
+    <header>
+      <p><strong>Alice Cieslik . Persönliche Website</strong> </p>
+
+      <p>Ich baue eine <strong>Website</strong>.</p>
+    </header>
+
+    <nav>
+      <strong>Navigation</strong> 
```

Comment:
As this is another heading I would suggest you to use the `h2` element instead of `strong`. :) I suppose that the user also knows that this part is the navigation. Therefore, I'm not sure if you really need to set a heading here.